ios/Sources/SettingFeature/NavigationBarConfigurator.swift

```diff
@@ -0,0 +1,53 @@
+import Styleguide
+import SwiftUI
+import UIKit
+
+public extension View {
+    func navigationBarColor(backgroundColor: Color, titleColor: Color) -> some View {
+        modifier(NavigationBarConfigurator(backgroundColor: backgroundColor, titleColor: titleColor))
+    }
+}
+
+fileprivate struct NavigationBarConfigurator: ViewModifier {
+    
+    let newBackgroundColor: Color
+    let newTitleColor: Color
+    
+    let previousBackgroundColor: Color
+    let previousTitleColor: Color
```

Comment:
IMO

I think type of these properties should be `Color?`.
If these properties are `nil`, it isn't needed to set `backgroundColor` and `foregroundColor` on Disappear 😄

Reply:
I see! I'll fix1

Reply:
I made these two variables nullable! please check whether it is okey for you.